client/view/Form/form.js

```diff
@@ -16,8 +16,12 @@ export function initForm() {
     if (!form.checkValidity()) {
       return
     }
+
     let formData = new FormData(form)
     let query = formData.get('query')
+    if(getUrlQuery() !== query) {
```

Comment:
Violate Prettier formatting rules.

You need to install Prettier to WebStorm or ask to add Prettier to `pre-commit` hook.